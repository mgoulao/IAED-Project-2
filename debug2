add 1 "ad" 1
add 2 "ad" 2
add 3 "ad" 3 1
add 4 "ad" 4 1 2
add 5 "ad" 5 3 4
add 7 "ad" 7 4
add 6 "ad" 6 4 7
duration
path
duration
add 9 "ad" 9 3
add 10 "ad" 10 9 5
duration
path
duration
exit