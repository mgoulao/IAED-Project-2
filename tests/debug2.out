1 "ad" 1
2 "ad" 2
3 "ad" 3 1
4 "ad" 4 1 2
5 "ad" 5 3 4
7 "ad" 7 4
6 "ad" 6 4 7
2 "ad" 2 [0 CRITICAL]
4 "ad" 4 [2 CRITICAL] 1 2
7 "ad" 7 [6 CRITICAL] 4
6 "ad" 6 [13 CRITICAL] 4 7
project duration = 19
1 "ad" 1 [0 1]
2 "ad" 2 [0 CRITICAL]
3 "ad" 3 [1 11] 1
4 "ad" 4 [2 CRITICAL] 1 2
5 "ad" 5 [6 14] 3 4
7 "ad" 7 [6 CRITICAL] 4
6 "ad" 6 [13 CRITICAL] 4 7
1 "ad" 1
2 "ad" 2
3 "ad" 3 1
4 "ad" 4 1 2
5 "ad" 5 3 4
7 "ad" 7 4
6 "ad" 6 4 7
9 "ad" 9 3
10 "ad" 10 9 5
1 "ad" 1 [0 CRITICAL]
3 "ad" 3 [1 CRITICAL] 1
9 "ad" 9 [4 CRITICAL] 3
10 "ad" 10 [13 CRITICAL] 9 5
project duration = 23
1 "ad" 1 [0 CRITICAL]
2 "ad" 2 [0 2]
3 "ad" 3 [1 CRITICAL] 1
4 "ad" 4 [2 4] 1 2
5 "ad" 5 [6 8] 3 4
7 "ad" 7 [6 10] 4
6 "ad" 6 [13 17] 4 7
9 "ad" 9 [4 CRITICAL] 3
10 "ad" 10 [13 CRITICAL] 9 5
